foundation/org.eclipse.persistence.extension/pom.xml

```diff
@@ -54,6 +54,11 @@
             <artifactId>junit</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>jakarta.json</groupId>
+            <artifactId>jakarta.json-api</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Why is dependency there? I don't see any JSON test in this Maven module.

Reply:
[ERROR] Errors: 
[ERROR]   SLF4JLoggerTest.testGetLevel » NoClassDefFound jakarta/json/JsonValue
[ERROR]   SLF4JLoggerTest.testLogExceptionStackTrace » NoClassDefFound jakarta/json/Json...
[ERROR]   SLF4JLoggerTest.testLogExceptionWithoutStackTrace » NoClassDefFound jakarta/js...
[ERROR]   SLF4JLoggerTest.testLogMessage » NoClassDefFound jakarta/json/JsonValue
[INFO] 
[ERROR] Tests run: 4, Failures: 0, Errors: 4, Skipped: 0